Assets/MXR.SDK/Runtime/Utils/Dispatcher.cs

```diff
@@ -0,0 +1,94 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using System;
+
+namespace MXR.SDK {
+    internal class Dispatcher : MonoBehaviour {
```

Comment:
Can you please add a top level comment as to what this class is for?

Reply:
Done in 5f86477c65649e14fc754ed0e1673ad39158b50b and also improved the class a bit by simplifying